packages/backend/src/app/database/migration/postgres/1700396157624-add-otp-entity.ts

```diff
@@ -0,0 +1,44 @@
+import { MigrationInterface, QueryRunner } from 'typeorm'
+import { logger } from '../../../helper/logger'
+
+export class AddOtpEntity1700396157624 implements MigrationInterface {
```

Comment:
You can avoid this by using JWT signing a token instead of storing it in the database.